gtk4/Gir.toml

```diff
@@ -1426,6 +1426,15 @@ status = "generate"
         [[object.function.parameter]]
         name = "pixbuf"
         nullable = false
+    [[object.function]]
+    name = "set_from_paintable"
+    rename = "set_paintable"
+    [[object.function]]
+    name = "set_from_icon_name"
+    rename = "set_icon_name"
+    [[object.function]]
+    name = "set_from_resource"
+    rename = "set_resource"
```

Comment:
The changes here looks good to me, although they should be in a separate commit + please add an inline comment to explain the reason behind the renaming.

Reply:
They are already in a second commit, but I expanded the commit message a bit and added a comment.